tests/_output/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
I just noted that we use two different patterns for storing empty directories within VCS.
_data/.gitkeep
_output/.gitignore

Reply:
It seems to be an inconsistency in Codeception - these files are autogenerated, when running the bootstrap command.

(I tried creating a new project and could reproduce it)